CS3219-AY2324S1/ay2324s1-course-assessment-g17
# Setup backend

Created the structure for `question_service`. 
Set up mongoose for mongodb and installed the following libraries. 
- Server: express
- Error handling & logging: http-errors, morgan
- dotenv
- Styling: eslint, prettier

# To test
- `cd backend`
- add `.env`
- `npm install`
- `npm start`
- you'll see the list of questions in `localhost:8080/api/questions`
